browser/resources/downloads/brave_overrides/index.ts

```diff
@@ -3,4 +3,6 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import 'chrome://resources/brave/br_elements/br_toolbar/br_toolbar.js'
```

Comment:
this is required as it was previously imported via `polymer_overriding.js` - I don't think this belongs in `lit_overriding.js`